website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,103 @@
+- **Global** - The single, outermost scope of Boundary that cannot be deleted.
+- **Org (Organization)** - Org scopes are contained in the global scope and are typically used to represent business units, organizations, or departments. Org scopes 
+are also utilized when there are multiple production-level auth methods that need to be kept in separate scopes.
+- **Project** - Project scopes are contained in Org scopes and contain infrastructure resources.
+
+#### Auth Methods
+[Auth Methods](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/auth-methods) are resources that provide a mechanism for users to authenticate to 
+Boundary. Auth methods can only be contained by the global or Org scope. The available auth methods are:
+- Username/Password
+- [OIDC](https://developer.hashicorp.com/boundary/tutorials/access-management/oidc-auth)
+
+#### Accounts
+[Accounts](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/accounts) are resources that represent a set of credentials used to establish the 
+identity of a user. An account is created with an auth method, and can be associated with a single user. Accounts can be contained by any scope.
+
+#### Users
+[Users](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/users) are resources that represent an individual person or entity for the purposes of 
+access control. A user can contain multiple accounts from multiple auth methods, and can be contained by any scope.
+
+#### Groups
+[Groups](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/groups) are resources that represent a collection of users, and can be contained by any 
+scope. [Managed groups](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/managed-groups) are a special kind of group that syncs with the auth 
+method's identity provider to provide up to date information.
+
+#### Roles
+[Roles](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/roles) are resources that contain a collection of 
+[permissions](https://developer.hashicorp.com/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles 
+can be contained by any scope, and can specify which scope the permissions will apply to.
+
+## Access Management
+#### Configuring Users
+When adding new users to Boundary using the password auth method, there are 3 steps to follow. Note: for OIDC auth methods, the following will be auto-generated using 
+information provided by the identity provider.
+1. **Auth Method** - Choose the scope and the auth method that the user will use to authenticate to Boundary. If the desired auth method does not exist, a new one needs 
+to be created.
+2. **Account** - After the auth method is selected, an account can be created under the auth method.
+3. **User** - In the same scope, create a new user, and then attach the new account to the new user. The user is now able to authenticate to Boundary.
```

Comment:
I think having the resource precede the step in the numbered list is a little confusing since it mixes talking about things with actions. I deleted them in my suggestion, but feel free to disregard if you think it's important info. Alternatively, we could put this info in a table with 2 columns listing the resource being created, and the step to create it.

Also, we (Education) are trying to move away from putting procedures and conceptual information in the same topic. I am wondering if longer term we might want to move this section into a standalone topic for **Configuring users** and build upon this.

Reply:
That makes sense, although I would like to bold the resource types in the steps